db/custom_seeds.rb

```diff
@@ -0,0 +1,22 @@
+I18n.available_locales.each do |locale|
```

Comment:
What do you think about adding a test for the seeds so we can be sure of not breaking them in the future when adding custom seeds?

Maybe a spec similar test to the one we have for `dev_seeds.rb` could work for that:

```
require "rails_helper"

describe "rake db:seed" do
  let :run_rake_task do
    Rake.application.invoke_task("db:seed")
  end

  it "seeds the database without errors" do
    expect { run_rake_task }.not_to raise_error
  end
end

```

Reply:
We should probably change `db:dev_seed` so it calls `db:seed` first :thinking:. I'm fine with the change, but I guess it's something to be done in the CONSUL main repo :thinking:.